[15.07] Release notes for 15.07.

jmchilton opened the pull request:
This includes a script to boot strap the notes for future releases, updated announcement page for 15.07, and new announcement page for 15.10.

Few points to consider during review:
- Feel free to push smaller stuff into my branch directly (open a PR if you don't have access yet, I'll add you).
- As always the ordering is somewhat arbitrary and reflects a John worldview of Galaxy, please start a conversation if you think other items should be highlighted instead.
- Please review 15.10 release schedule and comment (@natefoo in particular).
- The script used to bootstrap the release notes includes a list of github users that don't get explicit credit for their work (essentially old devteam). Depending on how you look at this - it could be seen as a special privilege for old devteam or for the newer committers. My thought on how to handle this is just let each person decide if they want credit or not in the release notes by modifying that list.

jmchilton committed:
Release notes for 15.07.

This includes a script to boot strap the notes for future releases, updated announcement page for 15.07, and new announcement page for 15.10.

martenson commented on doc/source/releases/15.07_announce.rst:
```diff
@@ -2,22 +2,60 @@
+  output collections.
 
-Schedule
+**Reports Application Enhancements**
```
I think this is not in 15.07

jmchilton replied:
Really? Ugh - is there a problem with my bootstrap script :(. I'll look into this.

jmchilton replied:
Yup - I don't understand what went wrong. Maybe there was some time between when Nate added the commit that created the branch and when it was merged back into dev.

natefoo commented on doc/source/releases/15.10_announce.rst:
```diff
@@ -0,0 +1,9 @@
+Schedule
+===========================================================
+ * Planned Freeze Date: October 5, 2015
+ * Planned Release Date: October 26, 2015
```
+1 on the schedule for 15.10

martenson commented on doc/source/releases/15.10_announce.rst:
```diff
@@ -0,0 +1,9 @@
+===========================================================
+October 2015 Galaxy Release (v 15.10)
```
the planned release was only in `dev` last time, if we put it in the `release_15.07' the latest actual release will never be on top

jmchilton committed:
Fixup end of release items for release notes.

jmchilton committed:
Typo noticed by @martenson.

martenson commented:
@jmchilton Do you want us to modify this or are you working on the script-generator?

jmchilton commented:
Going through and modifying by hand now. My WIP:

```
Deleting -

X 370,
X 410,
X 454,
X 462,
X 463,
X 472 
X 473
X 472-488
X 491-530,

X 532, 533, 536-546, 550, 553, 561, 565, 568, 569, 575, 577, 581-594

Adding - 528, 639, 649
```

jmchilton commented:
Okay - @martenson talked about the announcement not just being in dev and he is okay with that. I've fixed the release notes (replaced reports highlight with committers policy change). Removed the WIP tag.

nsoranzo commented on doc/source/releases/15.07_announce.rst:
```diff
@@ -2,22 +2,62 @@
+**Policies for Committers and Pull Requests**
+  The process for adding committers to the project and fielding pull requests has 
+  been formalized and documented in the source code. Three new committers have
+  added to the project - Björn Grüning, Nicola Soranzo, and Eric Rasche.
```
"have added" -> "have been added" ?

dannon committed:
Fix for explicit markup ending without a blank line

dannon committed:
Deprecation notice for cloudlaunch

jmchilton committed:
Merge pull request #9 from dannon/rn

Deprecation notice for cloudlaunch.

dannon committed:
Fix cloudlaunch link.

jmchilton committed:
Merge pull request #10 from dannon/rn

Fix cloudlaunch link.

natefoo commented:
:+1:

dannon commented:
+1
